### DISCUSSION TITLE
Running this on VMware?

### DISCUSSION THREAD

**Discussion by francisleemondia** [Microsoft/WSLab#614] [Ideas]
Hi so I’ve stumbled into this and really want to run this on my homelab which is a VMware environment with vcenter and a an ESXi host.  I’ve seen some other work (autolab) but hasn’t been updated in a while.  Is it possible that I can work with someone that’s willing to help me port this over? Or some tips or suggestions how to about running and or converting this to run in a VMware environment.

**Comment by Karl-WE** [comment]
Mslab is targeted for Hyper-V for a reason. It is possible to utilize PowerShell / PowerShell DSC to automate things on the host and within the VMs.

My question is then why you are looking into ESXi instead of Hyper-V.

Could you explain your use case? Eventually do you have an unused Server or Windows 11 PC?

With 64-128 GB RAM and 8 cores you can reach much already.
